src/terra-dev-site/guide/Releases.c.guide.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+## Release cycle
+
+* Weekly.
+  * Terra will release any component that has had changes in the last week. Generally we release mid week on Tuesday or Wednesday.
```

Comment:
nit: I'd update this to `Terra will release any component that has had changes reviewed and merged into master the last week`